openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -286,7 +286,11 @@ The hash algorithm SHA-256 MUST be supported by all the entities to generate and
 
 When using this profile alongside other hash algorithms, each entity SHOULD make it explicit in its metadata which other algorithms are supported.
 
-# Implementations Considerations
+# Implementation Considerations
+
+## Prerequisites
+
+This specification assumes certain prerequisites, including browser/OS support of certain features (for example the `haip://` custom URL scheme). This means some of the mandatory clauses might not be implementable for reasons outside the implementer's control.
```

Comment:
Should we mention both, custom URL scheme and DC API?
```suggestion
This specification assumes certain prerequisites, including browser/OS support of certain features (for example support for the `haip://` custom URL scheme or the Digital Credentials API within the browser). This means some of the mandatory clauses might not be implementable for reasons outside the implementer's control.
```

Reply:
I think there are two separate cases - if you can't do the DC API, you really can't do that whole flow at all - whereas you can support everything else in the redirect OID4VP without supporting `haip://`.

Reply:
then would it make sense not to mention custom url schemes entirely? 

```suggestion
Digital Credentials API related mandatory clauses in this specification depend on browser and/or operating system support. This means there might be environments beyond the implementer's control where these clauses cannot be implemented.
```